server/server.js

```diff
@@ -4,7 +4,7 @@ require('dotenv').config();
 const app = express();
 const bodyParser = require('body-parser');
 const sessionMiddleware = require('./modules/session-middleware');
-
+const UploaderS3Router = require('react-dropzone-s3-uploader/s3router');
```

Comment:
Hey @jakestucky  -- were you able to get this react-dropzone-s3-uploader working? 
I imagine you'll need some server-side endpoints for this, no?

I'd be interested in quick demo, if you have time 😄